libraries/AP_CANManager/AP_CANIfaceParams.cpp

```diff
@@ -35,6 +35,14 @@ const AP_Param::GroupInfo AP_CANManager::CANIface_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BITRATE", 2, AP_CANManager::CANIface_Params, _bitrate, 1000000),
 
+#if HAL_CANFD_SUPPORTED
+    // @Param: BITRATE
+    // @DisplayName: Bitrate of CAN interface
+    // @Description: Bit rate can be set up to from 10000 to 8000000
+    // @Range: 10000 8000000
```

Comment:
why would a user want 10000? Maybe set min to 1000000? Note that the @Range is recommended range, not absolute limit